### PR TITLE
Updated yaml file so client calls the correct API url.

### PR DESCRIPTION
**Pull request by matijaSos**

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -9,12 +9,14 @@ tasks:
     command: |
       export WASP_TELEMETRY_USER_ID=$GITPOD_WORKSPACE_ID
       cd MyNewApp
+      export REACT_APP_API_URL="https://3001-$GITPOD_WORKSPACE_ID.$GITPOD_WORKSPACE_CLUSTER_HOST"
```
Comment (axonasif) [inline .gitpod.yml]: This is neat! However we also have a command to get the same thing: `$(gp url 3001)`

**Comment by axonasif** [comment]
Hey @matijaSos, can you please enable this? I want to push a commit.
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

File: .gitpod.yml (hunk 1 above)
Comment (axonasif) [reply .gitpod.yml]: Also, I would recommend to these variables globally. Right now these env vars are scoped within the task terminal.

You could do this from a [custom dockerfile](https://www.gitpod.io/docs/config-docker#configure-a-custom-dockerfile):

I will push a commit.

**Comment by matijaSos** [comment]
@axonasif  sounds great, enabled! Although it seems like it has already been enabled, I just turned it off and back on.

**Comment by axonasif** [comment]
Thanks @matijaSos, but it seems that I still can't push 😅 
Maybe because the fork branch is `main`?

If you want, you can use the following:
- `.gitpod.yml`
```yaml
image:
  file: .gitpod.Dockerfile

# Learn more about this file at https://www.gitpod.io/docs/references/gitpod-yml
tasks:
  - command: |
      cd MyNewApp
      wasp start

ports:
  - port: 3000
    description: Web app
    onOpen: open-browser
  - port: 3001
    visibility: public
    description: Node.js server
    onOpen: ignore
```

- `.gitpod.Dockerfile`
```dockerfile
FROM gitpod/workspace-full:latest

ENV WASP_TELEMETRY_CONTEXT=gitpod

RUN curl -sSL https://get.wasp-lang.dev/installer.sh | sh \
	&& printf '%s\n' 'export WASP_TELEMETRY_USER_ID=$GITPOD_WORKSPACE_ID' \
		'export REACT_APP_API_URL="$(gp url 3001)"' > $HOME/.bashrc.d/700-wasp
```

**Comment by matijaSos** [comment]
Hm yeah not sure why is that @axonasif! Awesome, will use the code you provided. Where should I place `.gitpod.Dockerfile`, just next to `.gitpod.yml` I suppose?

**Comment by axonasif** [comment]
Yes!

**Pull request merged**